README.md

```diff
@@ -1,33 +1,64 @@
-@techchange/slack-webhooks
+console-to-slack
 ==========================
 
-Private NPM module for integrating with Slack webhooks. Currently supports overriding console.error and sending error messages to Slack. 
+NPM module for integrating with Slack webhooks. Currently supports overriding console.log, console.warn, and console.error to send messages to Slack.
 
 ## Installation
 ```
-npm install @techchange/slack-webhooks
+npm install console-to-slack
 ```
 
 ## Usage
 Include this module:
 ```javascript
-const tcSlackWebhooks = require('@techchange/slack-webhooks')(settings);
+const consoleToSlack = require('console-to-slack')(options);
```

Comment:
May want to either note that we're using ES6 syntax or switch to using `var`. I realize most people will know the difference, but might be good to be using more vanilla stuff in the docs.